tests/test_features/test_pcoords.py

```diff
@@ -92,6 +105,19 @@ def test_pcoords_sample_float(self):
         visualizer = ParallelCoordinates(sample=0.5)
         visualizer.fit_transform(self.X, self.y)
 
+    def test_pcoords_sample_float_shuffle(self):
+        """
+        Assert no errors occur using float 'sample' argument and shuffle, with different random_state args
+        """
+        visualizer = ParallelCoordinates(sample=0.5, shuffle=True)
+        visualizer.fit_transform(self.X, self.y)
+
+        visualizer = ParallelCoordinates(sample=0.5, shuffle=True, random_state=444)
+        visualizer.fit_transform(self.X, self.y)
+
+        visualizer = ParallelCoordinates(sample=0.5, shuffle=True, random_state=np.random.RandomState())
+        visualizer.fit_transform(self.X, self.y)
```

Comment:
Same comments as above in the int sample. 

Also, do we want to ensure no shuffling occurs when there is no sampling? Or is this satisfied in `test_static_subsample`?

Reply:
Added test_pcoords_sample_float_shuffle_false()

Tested in test_static_subsample.  I would argue this is the best place because X and y are exposed on input and output.